### ISSUE TITLE
Provide more email notifications

### ISSUE THREAD

**Issue by seav** [Cadasta/cadasta-platform#453]
Currently based on the code a user gets email at the following instances:
1. Email address confirmation during registration
2. Change of email address confirmation (although I can't seem to make this happen)
3. Password reset
4. Confirmation of getting added as an organization member (but only via the API)

I think the user should also get an email when they get assigned to a project, and probably also when the user gets deactivated or reactivated.

**Comment by amplifi** [comment]
We also have a datetime database field in `accounts_user` for `verify_email_by`, but aren't sending a follow-up to prompt for email address verification. I think we should consider implementing this feature.

**Comment by dpalomino** [comment]
Hi, I think probably this more a bug than a enhancement IMO. The behaviour should be coherent for both API and web UI....

**Comment by valaparthvi** [comment]
> I think the user should also get an email when they get assigned to a project, and probably also when the user gets deactivated or reactivated.
@seav @amplifi  I've currently added a bunch of notifications for user account activities( #1719 ) , should I add notification for account deactivation and reactivation as well?

**Comment by dpalomino** [comment]
@valaparthvi thanks!
> @seav @amplifi I've currently added a bunch of notifications for user account activities( #1719 ) , should I add notification for account deactivation and reactivation as well?

IMO that would be useful. @oliverroick @clash99, what do you think?

**Comment by amplifi** [comment]
@valaparthvi Yes, we should add notifications for deactivation & reactivation. Thanks!

**Comment by clash99** [comment]
I think we should add all the user notifications to part of the marketing "user communication" initiative so that we have a clear pictures of what triggers certain emails to be sent and the message contained in each.  I can add these two as well as the ones mentioned by @valaparthvi above.

**Comment by oliverroick** [comment]
Removing the "bug" label, it looks like this requires a bigger discussion about what emails we want to send and when.
